src/styles/styles.css

```diff
@@ -0,0 +1,306 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;600;700;800;900&display=swap');
+
+body {
+  width: 100%;
+  height: auto;
+  background-color: #f5f6fa;
+  text-decoration: none;
+}
+
+p {
+  margin-top: 0.4rem;
+  margin-bottom: 0;
+}
+
+#root {
+  width: 85%;
+  height: 70%;
+  margin: auto auto;
+  margin-top: 5%;
+  padding-bottom: 30rem;
+  border-radius: 4px;
+  box-shadow: 40px 60px 60px 0 rgba(0, 0, 0, 0.2);
+  background-color: #fafafa;
+}
+
+.navbar {
+  width: 100%;
+  height: 5.938rem;
+  background-color: #ffff;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.Bookstore-CMS,
+.categories-page {
+  width: 15rem;
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  padding-left: 1rem;
+  font-family: Montserrat Arial, Helvetica, sans-serif;
```

Comment:
- Nice use of `font-family` this way. I didn't know it. I appreciate it. Thanks. 👍
- Happy Coding 😄

Reply:
@Tufahel  Thanks for your feedback